python/rpdk/python/data/Python.gitignore

```diff
@@ -125,3 +125,5 @@ dmypy.json
 
 # contains credentials
 sam-tests/
+
+rpdk.log
```

Comment:
When the log file rotates it'll be `rpdk.log.1`, use a wildcard pattern here to fix that.